src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -5,26 +5,19 @@ sidebar_order: 3
 
 {% include learn-sdk.md platform="javascript" %}
 
-All our JavaScript-related SDKs provide the same API still there are some differences between them which this section of the docs explains.
-
-## Browser Compatibility
-
-Our JavaScript SDK supports all major browsers. In older browsers, error reports may have a degraded level of detail – for example, missing stack trace data or missing source code column numbers.
-
 The table below shows supported browsers:
 
 ![Sauce Test Status](https://saucelabs.com/browser-matrix/sentryio.svg)
 
 {% capture __alert %}
```

Comment:
I'd push this to the end, but link to it from the asterisk on IE10.  It's not a blocker to getting set up.

Reply:
@eurica I spoke with a JS dev about this table and they really liked it. They said it helps them in their decision making process of whether or not to use Sentry. Based on the conversation, it sounds like knowing what Internet Explorer versions are supported is a big weighing factor for them.

Reply:
@eurica Okay. I thought about it some more and linked the table down below Releases. :)